VKontakte/VKontakte/Model/News.swift

```diff
@@ -0,0 +1,14 @@
+// News.swift
+// Copyright © Dmi3. All rights reserved.
+
+import Foundation
+
+/// News Model
+struct News {
+    let userName: String
+    let avatarimageName: String
+    let newDate: String
+    let newsText: String
```

Comment:
```suggestion
    let newsText: String?
```
Некоторых полей может не быть

Reply:
хотел так. но если их не будет, мне кажется, удобней присвоить значение с ?? "". а так очень неудобные проверки и анрапы получаются